components/org.wso2.carbon.identity.oauth.common/src/main/java/org/wso2/carbon/identity/oauth/common/IDTokenResponseValidator.java

```diff
@@ -58,4 +62,24 @@ public void validateMethod(HttpServletRequest request) throws OAuthProblemExcept
     public void validateContentType(HttpServletRequest request) throws OAuthProblemException {
     }
 
+    /**
+     * Method to check whether the scope parameter string contains 'openid' as a scope.
+     *
+     * @param scope
+     * @return
+     */
+    private static boolean containOIDCScope(String scope) {
```

Comment:
Do we have a util package in oauth.common bundle? If so can we move this there as a public static method? We have similar one in OAuth2Util.isOIDCAuthzRequest(). However since this is in oauth bundle we can't take a dependency on this.